gulpfile.babel.js

```diff
@@ -261,27 +307,52 @@ gulp.task('deb:linux', ['package:linux'], (done) => {
   });
 });
 
+gulp.task('deb:linux:32', ['package:linux:32'], (done) => {
+  const debian = require('electron-installer-debian');
+
+  debian(_.extend({}, debianDefaults, {
+    src: `dist/${packageJSON.productName}-linux-ia32`,
+    arch: 'i386',
+  }), (err) => {
+    console.log('32bit deb package built');
+    if (err) return done(err);
+  });
+});
+
+gulp.task('deb:linux:64', ['package:linux:64'], (done) => {
+  const debian = require('electron-installer-debian');
+
+  debian(_.extend({}, debianDefaults, {
+    src: `dist/${packageJSON.productName}-linux-x64`,
+    arch: 'amd64',
+  }), (err) => {
+    console.log('64bit deb package built');
+    if (err) return done(err);
+    done();
+  });
+});
+
+const redhatDefaults = {
+  bin: packageJSON.productName,
+  dest: 'dist/installers/fedora',
+  depends: ['libappindicator1'],
+  maintainer: 'Samuel Attard <samuel.r.attard@gmail.com>',
+  homepage: 'http://www.googleplaymusicdesktopplayer.com',
+  icon: 'build/assets/img/main.png',
+  categories: ['AudioVideo', 'Audio'],
+};
+
 gulp.task('rpm:linux', ['package:linux'], (done) => {
```

Comment:
Added the separate linux builds as suggested here and it broke the CI builds, even though it reported a success.